pkg/cli/build.go

```diff
@@ -110,7 +110,7 @@ func addDockerfileFlag(cmd *cobra.Command) {
 }
 
 func addUseCogBaseImageFlag(cmd *cobra.Command) {
-	cmd.Flags().BoolVar(&buildUseCogBaseImage, "use-cog-base-image", false, "Use pre-built Cog base image for faster cold boots")
+	cmd.Flags().BoolVar(&buildUseCogBaseImage, "use-cog-base-image", true, "Use pre-built Cog base image for faster cold boots")
```

Comment:
Is the correct syntax for disabling this `--use-cog-base-image=false` or `--no-use-cog-base-image`?

Reply:
The former. `--no-` prefixes aren't natively supported. See https://github.com/spf13/pflag/issues/214

Reply:
That said, we could add a `--no-use-cog-base-image` flag alias